SQL-Query/Query-XII.(12)/SQLAnswer.md

```diff
@@ -0,0 +1,34 @@
+**12.Get all the appeasements in July month.<br>
+a.How do we differentiate between returns and appeasements?<br>
```

Comment:
Please also add the answer for this question.

Reply:
Added the answer, please review it once.